src/Services/PageService.cs

```diff
@@ -38,6 +40,7 @@ public PageService(ILocalSettingsService localSettingsService, IExperimentationS
         Configure<WhatsNewViewModel, WhatsNewPage>();
         Configure<ExtensionSettingsViewModel, ExtensionSettingsPage>();
         Configure<ExperimentalFeaturesViewModel, ExperimentalFeaturesPage>();
+        Configure<DeveloperFileExplorerViewModel, DeveloperFileExplorerPage>();

```

Comment:
This can change after https://github.com/microsoft/devhome/pull/2356 goes in, hopefully later this week.

Reply:
#2356 is checked in now.